### ISSUE TITLE
ISO

### ISSUE THREAD

**Issue by matt123491** [felixrieseberg/windows95#226]
Is it possible to get the ISO without installing the app? And if I do get the ISO can I use it for a VirtualBox?

**Comment by ST1NGRAY95** [comment]
I don't think so and even if it is it's best not to as the purpose of the app is to use it as a virtualizer. If you want an ISO image you may want to try https://winworldpc.com/product/windows-95/osr-2 (not an ad) or other websites online.

**Comment by hello-smile6** [comment]
@matt123491 Download a release `.zip` and get it that way.

**Comment by ST1NGRAY95** [comment]
> are you able to post the .iso image here? what app do you can you use?

You cannot post isos here as they are not allowed however you can go to the site I put in my above comment if you want. 

Could you reiterate your second question

**Comment by PJSRE** [comment]
My, how though..

**Comment by PJSRE** [comment]
I have the image now, but by any chance, have you used it with VurtualBox? 
https://forums.virtualbox.org/viewtopic.php?t=31895
https://virtualbox.org

**Comment by PJSRE** [comment]
> > are you able to post the .iso image here? what app do you can you use?
> 
> You cannot post isos here as they are not allowed however you can go to the site I put in my above comment if you want.
> 
> Could you reiterate your second question

I as in a application that can “install” iso images (not to the computer)

**Comment by hello-smile6** [comment]
> I have the image now, but by any chance, have you used it with VurtualBox? https://forums.virtualbox.org/viewtopic.php?t=31895 https://virtualbox.org

Use qemu.

**Comment by ST1NGRAY95** [comment]
> I have the image now, but by any chance, have you used it with VurtualBox? https://forums.virtualbox.org/viewtopic.php?t=31895 https://virtualbox.org

You only need to use a virtual machine if you are really serious however most people just use it for general nostalgia and fun

**Comment by hello-smile6** [comment]
> > I have the image now, but by any chance, have you used it with VurtualBox? https://forums.virtualbox.org/viewtopic.php?t=31895 https://virtualbox.org
> 
> You only need to use a virtual machine if you are really serious however most people just use it for general nostalgia and fun

https://copy.sh/v86

**Comment by felixrieseberg** [comment]
There's no ISO — Windows 95 shipped before bootable CDs were standard, and what this app uses is a raw hard-disk image (`windows95.img`). You can see it via Machine → Go to Disk Image in the app. For VirtualBox, convert it: `VBoxManage convertfromraw windows95.img windows95.vdi --format VDI`. There are also working QEMU configs in [docs/qemu.md](https://github.com/felixrieseberg/windows95/blob/main/docs/qemu.md). I don't distribute the image outside the app, so grabbing it from a release is the way.
